sdk/eventhub/event-hubs/CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 # Release History
 
-## 5.2.0 (Unreleased)
+## 5.2.0 (2020-05-05)
 
 - Updates the `EventHubProducerClient.sendBatch` API to accept an array of events.
-- Adds compatiblity with TypeScript versions 3.1 through 3.6+
+  ([PR #8622](https://github.com/Azure/azure-sdk-for-js/pull/8622))
```

Comment:
This is unfair..
event-hubs gets to keep the hashtag in [PR #num] in the changelog while the service-bus doesn't 😒

Reply:
It's all in the name of 'consistency'

Reply:
"We made this mistake before...."